index.rst

```diff
@@ -47,6 +48,7 @@ Team
 Basic information about the LSST Data Management Subsystem and our culture.
 Learn more about the Data Management Subsystem in :ldm:`294`.
 
+- :doc:`team/quiet`
```

Comment:
I'd put this at the bottom, rather than the top of these lists — the “quiet day” is important, but it is not more important than things like the code of conduct.

Reply:
ok